package.json

```diff
@@ -50,6 +50,7 @@
     "eslint-config-prettier": "^8.7.0",
     "eslint-plugin-prettier": "^4.2.1",
     "mocha": "^10.2.0",
+    "multer": "^1.4.5-lts.1",
```

Comment:
are we using `multer`?

Reply:
we are not, but TSOA is using for `@UploadedFile()` descriptor. It utilizes in a router.ts

P.S the below is generated by TSOA
```ts
// @ts-ignore - no great way to install types from subpackage
const promiseAny = require('promise.any');
import type { RequestHandler } from 'express';
import * as express from 'express';
const multer = require('multer');
const upload = multer();

        app.post('/attachment',
            authenticateMiddleware([{"bearerAuth":[]}]),
            upload.single('file'),
            ...(fetchMiddlewares<RequestHandler>(attachment)),
            ...(fetchMiddlewares<RequestHandler>(attachment.prototype.create)),
...
```

Reply:
ok so this should be a dpendency not a devdependency

Reply:
this should have been updated now